lib/Utils/Graph/Graph.h

```diff
@@ -148,10 +218,109 @@ class Graph {
     return output;
   }
 
+  /* get vertices in graph with 0 incoming edges */
```

Comment:
you can use `llvm::copy_if(vertices, [V& v vertex] { return getInEdges(v) == 0; })` if you want!